src/cli.c

```diff
@@ -57,7 +57,7 @@ int main(int argc, char *argv[])
     bool spirvSource = false;
     ShaderCross_ShaderFormat destinationFormat = SHADERFORMAT_INVALID;
     SDL_ShaderCross_ShaderStage shaderStage = SDL_SHADERCROSS_SHADERSTAGE_VERTEX;
-    SDL_ShaderCross_ShaderModel shaderModel;
+    SDL_ShaderCross_ShaderModel shaderModel = 0;
```

Comment:
That's `SDL_SHADERCROSS_SHADERMODEL_5_0`.
Is this ok, or should the enum contain an "invalid" shadermodel?

Reply:
I'll add an invalid entry.

Reply:
Reading over the code, won't this invalid shader model cause a null pointer dereference?

Reply:
Nevermind, it's ok.

Reply:
I don't think so. It just gets passed as a parameter to TranspileHLSLFromSPIRV which sets a numeric value for SPIRV-Cross in an if-else. Are you perhaps confusing shader model for shader stage?

Reply:
Oh, I see. We should null guard the output of that function.

Reply:
I'm following the `shaderModel` variable in `cli.c`.
When no `-m` is passed it is invalid.
Functions that use `shaderModel` will return `NULL`, which will be written to the output.